supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -10,6 +10,35 @@ Follow _The IBM Style Guide_  guidance of _day Month year_ for date formats, for
 When the format _day Month year_ causes a presentation or clarity issue, use _Month day, year_ (for example, October 3, 2019) instead.
 
 
+[[conscious-language]]
+== Conscious language
+
+The Conscious Language Group is supporting the Red Hat commitment to remove problematic language from our code, documentation, websites, and open source projects that Red Hat is involved with.
+ 
+.blacklist and whitelist 
```

Comment:
I feel like this heading should be at a higher level than the examples below. Maybe consider making it a real level 3 heading (`=== blacklist and whitelist`) so that it can be seen as an umbrella for all the examples that fall below it?